### ISSUE TITLE
Show relevant fields when issue or flag used in filter

### ISSUE THREAD

**Issue by tucotuco** [gbif/hosted-portals#13]
That fact that I am suggesting a feature like this shows just how awesome the hosted portals are already.

To facilitate the investigations and correction of data issues, it would be extremely useful to show the fields relevant to the issue used in the filter. That would let a user readily see and browse, for example, invalid datums without having to download them.

**Comment by MortenHofft** [comment]
Do I understand correctly when I read this issue as:

* it would be useful to know what fields affect issue X
* it would be useful to configure the table to show the relevant columns (if not even adding them automatically when searching for issues)

**Comment by tucotuco** [comment]
@MortenHofft Yes. I was suggesting to add them to the table when the flag is part of the query. I wasn't suggesting that any other indication of what fields are involved, but it would be great to see descriptions of the flags that revealed that information in a description property like that for other columns in the table.
